src/MaxMind/Db/Reader.php

```diff
@@ -314,7 +342,7 @@ private function findMetadataStart(string $filename): int
                 break;
             }
 
-            $value = fread($handle, $markerLength);
+            $value = fread($handle, abs($markerLength));
```

Comment:
I think it would be better to add a type-hint on `$METADATA_START_MARKER` rather than calling `abs` repeatedly.

Reply:
Sure, I'll go ahead and change this to a type-hint with a comment.

Reply:
Done. There is a similar error in Util.php, which I'll correct as well.